doc/lua_api.txt

```diff
@@ -617,6 +617,18 @@ A nodebox is defined as any of:
         wall_bottom = box,
         wall_side = box
     }
+    {
+        -- A node that has optional boxes depending on neighbouring nodes'
+        -- presence and type. See also `connects_to`.
+        type = "connected",
+        fixed = box OR {box1, box2, ...}
+        connect_up = box OR {box1, box2, ...}
+        connect_down = box OR {box1, box2, ...}
+        connect_n = box OR {box1, box2, ...}
+        connect_e = box OR {box1, box2, ...}
+        connect_w = box OR {box1, box2, ...}
+        connect_s = box OR {box1, box2, ...}
```

Comment:
front/back/left/right are relative, not absolute, which seemed more confusing to me. N = +Z, yes, just what it says in the game if you press F5.